### ISSUE TITLE
Windows 2016 runners being removed in March 2022

### ISSUE THREAD

**Issue by nschonni** [sass/node-sass#3212]
https://github.blog/changelog/2021-10-19-github-actions-the-windows-2016-runner-image-will-be-removed-from-github-hosted-runners-on-march-15-2022/
We currently use them for Node 12 and 14 builds since those platforms supported a lower minimum supported version. We could just bump them to latest or 2019 though.

**Comment by xzyfer** [comment]
Yeah we could just bump them. The release binaries are still being generated by AppVeyor atm.

**Comment by MikeMcC399** [comment]
@nschonni 
Today is the deadline which you mentioned for the removal of the Windows 2016 hosted runner.

https://github.com/actions/virtual-environments/blob/main/images/win/Windows2016-Readme.md also lists today, March 15, 2022 and it points to issue `4312` in https://github.com/actions/virtual-environments/issues.

**Comment by xzyfer** [comment]
This deadline has passed and the respective github actions are now failing.

**Comment by xzyfer** [comment]
We might as well ship this with https://github.com/sass/node-sass/issues/3250 and https://github.com/sass/node-sass/pull/3249 in the new major.

**Comment by MikeMcC399** [comment]
According to issue `5238` in https://github.com/actions/virtual-environments/issues/ the change was delayed until April 1, 2022, however also that deadline has now passed.
